src/plugins/dashboard/public/application/actions/clone_panel_action.tsx

```diff
@@ -39,7 +39,7 @@ export interface ClonePanelActionContext {
 export class ClonePanelAction implements ActionByType<typeof ACTION_CLONE_PANEL> {
   public readonly type = ACTION_CLONE_PANEL;
   public readonly id = ACTION_CLONE_PANEL;
-  public order = 11;
+  public order = 45;
```

Comment:
Adjusted to follow: https://github.com/elastic/kibana/issues/60138#issuecomment-617860600